recipes/y-websocket/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "y-websocket" %}
```

Comment:
this clobbers y-js/y-websocket whcih is a nodejs package ...

Reply:
Right, I renamed the package `ypy-websocket`.